src/mixins/segmentation/freehandSegmentationMixin.js

```diff
@@ -0,0 +1,189 @@
+import external from '../../externalModules.js';
+import store from '../../store';
+import { getLogger } from '../../util/logger.js';
+import { draw, drawJoinedLines, getNewContext } from '../../drawing';
+
+const logger = getLogger('tools:ScissorsTool');
+
+const { getters, setters } = store.modules.segmentation;
+
+/**
+ * Render hook: draws the FreehandScissors's outline
+ *
+ * @param {Object} evt Cornerstone.event#cornerstoneimagerendered > cornerstoneimagerendered event
+ * @memberof Tools.ScissorsTool
+ * @returns {void}
+ */
+function renderToolData(evt) {
+  const eventData = evt.detail;
+  const { element } = eventData;
+  const color = getters.brushColor(element, true);
+  const context = getNewContext(eventData.canvasContext.canvas);
+  const handles = this.handles;
+
+  draw(context, context => {
+    if (handles.points.length > 1) {
+      for (let j = 0; j < handles.points.length; j++) {
```

Comment:
Would this loop even run if `points.length` were `0` or less? May not need first conditional.

Reply:
I didn't write this, but looks like you need at least 2 points to draw the line in the draw loop.

However, this shouldn't be _inside_ the draw call, i'll add this before:

if (points.length < 2) {
  return;
}